src/plugins/intel_cpu/src/nodes/reorder.cpp

```diff
@@ -386,18 +372,18 @@ void Reorder::setDynamicBatchLim(int lim) {
     if (prim) {
         auto &dstMemPtr = getChildEdgeAt(0)->getMemoryPtr();
         auto &srcMemPtr = getParentEdgeAt(0)->getMemoryPtr();
-        memory::desc src_d = srcMemPtr->GetDescWithType<DnnlMemoryDesc>()->getDnnlDesc();
-        memory::desc dst_d = dstMemPtr->GetDescWithType<DnnlMemoryDesc>()->getDnnlDesc();
-        void *src_data_hdl = srcMemPtr->GetData();
-        void *dst_data_hdl = dstMemPtr->GetData();
+        DnnlMemoryDescPtr srcMemDescPtr = srcMemPtr->GetDescWithType<DnnlMemoryDesc>();
+        DnnlMemoryDescPtr dstMemDescPtr = dstMemPtr->GetDescWithType<DnnlMemoryDesc>();
+        memory::desc src_d = srcMemDescPtr->getDnnlDesc();
+        memory::desc dst_d = dstMemDescPtr->getDnnlDesc();
 
         src_d.data.dims[0] = batchToProcess();
         src_d.data.padded_dims[0] = batchToProcess();
 
         dst_d.data.dims[0] = batchToProcess();
         dst_d.data.padded_dims[0] = batchToProcess();
 
-        createReorderPrimitive(src_d, src_data_hdl, dst_d, dst_data_hdl);
+        createReorderPrimitive(srcMemDescPtr, dstMemDescPtr);
```

Comment:
That would be more safe to create new descriptors form the changed `memory::desc` rather than use the existing DnnlMemoryDescPtr with unmodified shape.
```cpp
srcMemDescPtr = DnnlExtensionUtils::makeDescriptor(src_d);
dstMemDescPtr = DnnlExtensionUtils::makeDescriptor(dst_d);
```

Reply:
done, thank you!